gadget-container/gadgettracermanager/main.go

```diff
@@ -340,7 +347,10 @@ func main() {
 
 		stringBufferLength := os.Getenv("EVENTS_BUFFER_LENGTH")
 		if stringBufferLength == "" {
-			log.Fatalf("Environment variable EVENTS_BUFFER_LENGTH not set")
+			stringBufferLength = config.Config.GetString(gadgettracermanagerconfig.EventsBufferLengthKey)
+		}
+		if stringBufferLength == "" {
+			log.Fatalf("Environment variable EVENTS_BUFFER_LENGTH or config not set")
```

Comment:
What's the reason to keep the env variables? Compatibility? 

I think it's fine to keep them, but:
1. The configuration file should have precedence 
2. We should print warnings indicating those variables are deprecated, and remove them later on.